docker/build/Dockerfile

```
@@ -22,6 +22,15 @@ COPY --from=golangci/golangci-lint:v1.50 /usr/bin/golangci-lint /usr/local/bin/
 RUN wget -O /usr/local/bin/kind https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64 \
     && chmod +x /usr/local/bin/kind
 
+# adding crd documentation tool here, since docs-build does not have golang base to create binary from go file
+RUN wget -O gen-crd-api-reference-docs_linux_amd64.tar.gz https://github.com/ahmetb/gen-crd-api-reference-docs/archive/refs/tags/v0.3.0.tar.gz \
+    && tar xzfv gen-crd-api-reference-docs_linux_amd64.tar.gz  \
+    && mkdir crd_documentation \
+    && cp -r gen-crd-api-reference-docs-0.3.0/* ./crd_documentation \
+    && cd crd_documentation \
+    && go build \
+    && cp -r * /usr/local/bin/
```

@Kartik-Garg we can achieve the same using this command. Could you please try?

```suggestion
RUN GOBIN=/usr/local/bin go install github.com/ahmetb/gen-crd-api-reference-docs@v0.3.0
```

---

@PrasadG193 it just gets the executable binary, but not the other required components like `config file` and `template folder` .

---

Do we need other components? Looks like we are only copying the binary, right?

---

If we just need a couple of files, lets maintain local copy instead of cloning whole repo just for this.

---

@PrasadG193 we do need just two other components, i.e. `config file` and the `template folder` .
Sure I will add a local copy of those and just download the binary using `RUN GOBIN` command.